carta/session.py

```diff
@@ -427,11 +427,11 @@ def set_viewer_mode(self, panel_mode):
         Parameters
         ----------
         panel_mode : {0}
-            To adopt single-panel mode (``0``) or multiple-panel mode (``1``).
+            To adopt single-panel mode (:obj:`carta.constants.PanelMode.SINGLE`) or multiple-panel mode (:obj:`carta.constants.PanelMode.MULTIPLE`).
```

Comment:
Actually, because the enum will be linked from the type description, it isn't necessary to list the options here either. Also, the description should be a noun phrase. So I would suggest just `The panel mode to adopt.`

Reply:
Done in dec7725.